kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/inspectors/InspectorAliases.kt

```diff
@@ -1,79 +1,100 @@
 package io.kotest.inspectors
 
 /** Alias for [Sequence.forAll] */
-fun <T> Sequence<T>.shouldForAll(fn: (T) -> Unit) = forAll(fn)
+inline fun <T> Sequence<T>.shouldForAll(fn: (T) -> Unit) = forAll(fn)
+
 /** Alias for [Array.forAll] */
-fun <T> Array<T>.shouldForAll(fn: (T) -> Unit) = forAll(fn)
+inline fun <T> Array<T>.shouldForAll(fn: (T) -> Unit) = forAll(fn)
+
 /** Alias for [Collection.forAll] */
-fun <T> Collection<T>.shouldForAll(fn: (T) -> Unit) = forAll(fn)
+inline fun <T> Collection<T>.shouldForAll(fn: (T) -> Unit) = forAll(fn)
 
 
 /** Alias for [Sequence.forOne] */
-fun <T> Sequence<T>.shouldForOne(fn: (T) -> Unit) = forOne(fn)
+inline fun <T> Sequence<T>.shouldForOne(fn: (T) -> Unit) = forOne(fn)
+
 /** Alias for [Array.forOne] */
-fun <T> Array<T>.shouldForOne(fn: (T) -> Unit) = forOne(fn)
+inline fun <T> Array<T>.shouldForOne(fn: (T) -> Unit) = forOne(fn)
+
 /** Alias for [Collection.forOne] */
-fun <T> Collection<T>.shouldForOne(fn: (T) -> Unit) = forOne(fn)
+inline fun <T> Collection<T>.shouldForOne(fn: (T) -> Unit) = forOne(fn)
 
 
 /** Alias for [Sequence.forExactly] */
-fun <T> Sequence<T>.shouldForExactly(k: Int, fn: (T) -> Unit) = forExactly(k, fn)
+inline fun <T> Sequence<T>.shouldForExactly(k: Int, fn: (T) -> Unit) = forExactly(k, fn)
+
 /** Alias for [Array.forExactly] */
-fun <T> Array<T>.shouldForExactly(k: Int, fn: (T) -> Unit) = forExactly(k, fn)
+inline fun <T> Array<T>.shouldForExactly(k: Int, fn: (T) -> Unit) = forExactly(k, fn)
+
 /** Alias for [Collection.forExactly] */
-fun <T> Collection<T>.shouldForExactly(k: Int, fn: (T) -> Unit) = forExactly(k, fn)
+inline fun <T> Collection<T>.shouldForExactly(k: Int, fn: (T) -> Unit) = forExactly(k, fn)
 
 
 /** Alias for [Sequence.forSome] */
-fun <T> Sequence<T>.shouldForSome(fn: (T) -> Unit) = forSome(fn)
+inline fun <T> Sequence<T>.shouldForSome(fn: (T) -> Unit) = forSome(fn)
+
 /** Alias for [Array.forSome] */
-fun <T> Array<T>.shouldForSome(fn: (T) -> Unit) = forSome(fn)
+inline fun <T> Array<T>.shouldForSome(fn: (T) -> Unit) = forSome(fn)
+
 /** Alias for [Collection.forSome] */
-fun <T> Collection<T>.shouldForSome(fn: (T) -> Unit) = forSome(fn)
+inline fun <T> Collection<T>.shouldForSome(fn: (T) -> Unit) = forSome(fn)
 
 
 /** Alias for [Sequence.forAny] */
-fun <T> Sequence<T>.shouldForAny(fn: (T) -> Unit) = forAny(fn)
+inline fun <T> Sequence<T>.shouldForAny(fn: (T) -> Unit) = forAny(fn)
```

Comment:
It seems redundant to have both `forSome` and `forAny`. What am I missing?

Reply:
I think `forSome` means strictly "At least one but not all" and `forAny` means "For 1, 2 or N inclusive`